docs/src/modules/components/MarkdownElement.js

```diff
@@ -230,8 +230,10 @@ const Root = styled('div')(({ theme }) => ({
     border: '1px solid',
     borderRadius: theme.shape.borderRadius,
     '& > p': {
-      margin: 0,
       color: 'inherit',
+      '&:last-child': {
```

Comment:
Fix the broken margin when there are some

**Before**
<img width="740" alt="Screenshot 2022-05-31 at 01 54 05" src="https://user-images.githubusercontent.com/3165635/171069198-9a2a40d0-973a-4353-83d6-1559bf2473f7.png">

**After**
<img width="735" alt="Screenshot 2022-05-31 at 01 53 44" src="https://user-images.githubusercontent.com/3165635/171069163-703af8c8-8b55-4ceb-aaca-25f6f7d7e7d4.png">

cc @siriwatknp